go/tasks/plugins/array/k8s/monitor.go

```diff
@@ -112,6 +112,7 @@ func LaunchAndCheckSubTasksState(ctx context.Context, tCtx core.TaskExecutionCon
 
 		var monitorResult MonitorResult
 		monitorResult, err = task.Monitor(ctx, tCtx, kubeClient, dataStore, outputPrefix, baseOutputDataSandbox)
+		logLinks = task.LogLinks
```

Comment:
needed?

Reply:
yes even though everything i read indicates golang slices are effectively pass by reference this failed on sandbox deployment unless i updated